src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -665,6 +665,8 @@ private BoundExpression BindExpressionInternal(ExpressionSyntax node, BindingDia
 
                 case SyntaxKind.NumericLiteralExpression:
                 case SyntaxKind.StringLiteralExpression:
+                case SyntaxKind.SingleLineRawStringLiteralExpression:
```

Comment:
One more usage of `SyntaxKind.StringLiteralExpression` occurs in `FindAppropriateRangeWorker`. Should it be updated to handle the new kinds too?

Reply:
i will check!

Reply:
I'm a bit confused.  FindAppropriateRangeWorker is in the IDE layer.  That won't be updated as part of this PR.

Reply:
That's fine. I assume you'll do a pass looking for places that reference `SyntaxKind.StringLiteralExpression` when you get to IDE layer then.

Reply:
Oh yes.  Definitely :)